source/about/staff.rst

```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
 |                                   |                                    |
 |                                   | Website Manager                    |
 +-----------------------------------+------------------------------------+
+| phase                             | Ore Back-end Manager               |
++-----------------------------------+------------------------------------+
 | phroa                             | IRC Manager                        |
 +-----------------------------------+------------------------------------+
 | progwml6                          | Ore Back-end Manager               |
 |                                   |                                    |
-|                                   | Systems Management                 |
+|                                   | Systems Manager                    |
```

Comment:
Maybe `System Manager`, but I'm not sure.

Reply:
I'm just mirroring the phrasing from lukegb's position.